Source/WebKit/UIProcess/_WKTouchEventGenerator.mm

```diff
@@ -317,14 +312,8 @@ - (void)liftUpAtPoints:(CGPoint*)rawLocations touchCount:(NSUInteger)touchCount
 
 - (void)liftUp:(CGPoint)location touchCount:(NSUInteger)touchCount window:(UIWindow *)window
 {
-    touchCount = std::min(touchCount, HIDMaxTouchCount);
-
-    Vector<CGPoint> locations(touchCount);
-
-    for (NSUInteger index = 0; index < touchCount; ++index)
-        locations[index] = location;
-    
-    [self liftUpAtPoints:locations.data() touchCount:touchCount window:window];
+    Vector<CGPoint> locations(std::min(touchCount, HIDMaxTouchCount), location);
```

Comment:
Should we write `Vector<CGPoint, HIDMaxTouchCount>` to avoid heap memory allocation here?

Reply:
Or just use FixedVector?

Reply:
Oh, Actually, I don't think FixedVector would avoid the heap allocation. My bad.